src/QsCompiler/CompilationManager/AssemblyLoader.cs

```diff
@@ -25,6 +27,12 @@ namespace Microsoft.Quantum.QsCompiler
     /// </summary>
     public static class AssemblyLoader
     {
+        public enum SyntaxTreeSerializationProtocol
```

Comment:
Hm, I would love to eliminate this from the public API; while we will want to keep the Newtonsoft infrastructure for backwards compatibility, I also think we should deprecate it in the near future (though we should wait quite a while before we actually remove it).

Reply:
I removed this and now the backwards compatibility with Newtonsoft is just encapsulated in the `FromResource` private method.